addons/l10n_de/report/din5008_report.xml

```diff
@@ -95,9 +95,10 @@
                             </td>
                         </tr>
                         <tr t-if="o and 'l10n_de_addresses' in o">
+                            <t t-set="colspan_val" t-value="2 if len(o.l10n_de_addresses) == 1  else 1"/>
```

Comment:
I don't quite understand this.
If I print without any shipping address, so just an invoicing address, I have it in the middle right. Is it normal? (I suppose it comes from here or the float-right below)
![image](https://github.com/odoo/odoo/assets/93133576/a368e9c6-26d9-4b1f-b46e-37cfb3bc0bfb)

Reply:
For that it's more of where the address must be placed, since the shipping address is empty for the moment there is an empty section with a title, but since it should be totally invisible i kept the placement of the invoicing address like the other section was invisible.
I've check multiple examples on internet and it was like that for most of them. Now we can maybe check with a partner if that's really ok